lib/improver/tests/nowcasting/optical_flow/test_OpticalFlow.py

```diff
@@ -651,16 +645,30 @@ def test_basic(self):
 
     def test_values(self):
         """Test velocity values are as expected (in m/s)"""
-        ucube, vcube = self.plugin.process(self.cube1, self.cube2)
-        self.assertAlmostEqual(np.mean(ucube.data), -2.12078369915)
-        self.assertAlmostEqual(np.mean(vcube.data), 2.12078369915)
+        ucube, vcube = self.plugin.process(self.cube1, self.cube2, boxsize=3)
+        self.assertAlmostEqual(np.mean(ucube.data), -2.171908358)
+        self.assertAlmostEqual(np.mean(vcube.data), 2.171908358)
+
+    def test_update_smoothing_radius(self):
+        """Test data smoothing radius is updated if cube time difference is not
+        15 minutes"""
+        time_unit = self.cube2.coord("time").units
+        new_time = time_unit.num2date(self.cube2.coord("time").points[0])
+        new_time += datetime.timedelta(seconds=900)
+        self.cube2.remove_coord("time")
+        time_coord = DimCoord(time_unit.date2num(new_time),
+                              standard_name="time", units=time_unit)
+        self.cube2.add_aux_coord(time_coord)
+        with self.assertRaises(ValueError):
```

Comment:
It would be good to actually catch the error message using `assertRaisesRegexp` that is being generated here, so that it's clearer which ValueError you're trying to catch at this point.

Reply:
I'm not really testing for that value error here - I just can't make a test for the data smoothing radius adjustment without tripping something.